pkg/comment/comment_message_loader_test.go

```diff
@@ -0,0 +1,348 @@
+package comment_test
+
+import (
+	"github.com/arquillian/ike-prow-plugins/pkg/comment"
+	"github.com/arquillian/ike-prow-plugins/pkg/config"
+	. "github.com/arquillian/ike-prow-plugins/pkg/internal/test"
+	"github.com/arquillian/ike-prow-plugins/pkg/scm"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"gopkg.in/h2non/gock.v1"
+)
+
+const ProwPluginName string = "my-test-plugin"
+
+var _ = Describe("Comment message creation", func() {
+
+	Context("Creation of default comment messages that are sent to a validated PR when custom message file is not set", func() {
+
+		It("should create default message referencing to documentation when url to config is empty", func() {
+			// given
+			conf := config.PluginConfiguration{PluginName: ProwPluginName}
+			comment := &comment.Comment{
+				Message: &comment.Message{ConfigFile: conf.LocationURL, Documentation: "#_my_test_plugin"},
+			}
+
+			// when
+			msg := comment.LoadMessage(conf, scm.RepositoryChange{})
+
+			// then
+			Expect(msg).To(ContainSubstring("http://arquillian.org/ike-prow-plugins/#_my_test_plugin"))
```

Comment:
verify that it doesn't contain the string `Your plugin configuration is stored in the` - opposite to this https://github.com/arquillian/ike-prow-plugins/pull/171/files#diff-5a1174093f81d3fdc9e33c0c3727465dR46